test/runTest/store.js

```diff
@@ -92,7 +93,8 @@ function parseRunHash(str) {
     return {
         expectedVersion: parts[0],
         actualVersion: parts[1],
-        renderer: parts[2]
+        renderer: parts[2],
+        useCoarsePointer: parts[3]
```

Comment:
Whether this new hash part should be one of the conditions that check the store version.
https://github.com/apache/echarts/blob/c1c9faf2691ff94231e8019f375ac8c210b2b5e8/test/runTest/store.js#L111-L117